guide/onboarding/automatic-cloud-backup.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: guide
+title: Automatic cloud backup
+description: How to backup a recovery phrase automatically
+grand_parent: Onboarding
+parent: Backing up a recovery phrase
+nav_order: 1
+permalink: /guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/
+main_classes: -no-top-padding
+---
+
+# Automatic cloud backup
+
+### How it Works
+
+The user’s key is encrypted and backed up on their respective device/cloud storage provider. In the case that their device is lost, stolen, or broken, the user can simply log into their Apple or Google account to regain access to their funds. To learn more about the technical details, hop over to the [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) section on this topic.
+
+> Crypto rookies perceive themselves as “incapable of self-managing keys”.
+>
+> <cite>As researched by <a href="https://voskart.de/pdf/u_in_crypto.pdf">Abramova et al</a></cite>
+
+This method provides users with the feeling that they can rely on their cloud provider to regain access to their funds rather than putting the responsibility on themselves.
+
+Below is an example of what the user interface might look like for an automatic cloud backup that significantly reduces the friction during onboarding.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-intro.png"
+   retina = "/assets/images/guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/cloud-backup-intro@2x.png"
+   alt-text = "Screen explaining automatic cloud backups"
+   width = 250
+   height = 541
```

Comment:
You can add `class = "shadow"` to these images if you want the drop shadow effect for a bit more contrast.

Reply:
Captions would also be great for these to explain the purpose of each screen.

Reply:
Sorry, should be `layout = "shadow"`.